src/models/Client.ts

```diff
@@ -2,6 +2,7 @@ import mongoose from 'mongoose';
 
 export type ClientDocument = mongoose.Document & {
     sub: string;
+    name: string;
```

Comment:
@GarfDev We had discussed storing the client name in the API's clients collection. I was unable to find your work for that in your PR and started with that implementation myself, hence the extended model over here.